cpan/libnet/t/hostname.t

```diff
@@ -36,7 +36,7 @@ else {
  print "not ok 1\n";
 }
 
-# This checks thats hostanme does not overwrite $_
+# This checks that hostname does not overwrite $_
```

Comment:
This change is in an unrelated distribution which is maintained on CPAN and should not be included.